sds_data_manager/stacks/backup_bucket_stack.py

```diff
@@ -0,0 +1,96 @@
+from aws_cdk import (
+    Environment,
+    RemovalPolicy,
+    Stack,
+)
+from aws_cdk import (
+    aws_iam as iam,
+)
+from aws_cdk import (
+    aws_s3 as s3,
+)
+from constructs import Construct
+
+
+class BackupBucket(Stack):
+    """
+    This stack creates the destination bucket for data backups. It can be run in
+    the same account as SdsDataManager, or in a separate account.
+    The source_account is a required parameter. This source account should be the
+    AWS account for the source bucket.
+
+    For replication to work, you also need to deploy SdsDataManager and create
+    the source bucket and replication role. Then, you need to manually update
+    the role_arn variable with the replication role created.
```

Comment:
I can look into this, but I don't think it will work, because the sds_data_manager is created in a different account. When  deploying the backup bucket, it won't be able to see the sds_data_manager in the dev account.

Reply:
I think there is way to add this dependency at very high level like in `app.py` but we can look at it post SIT-2.